Authorize.Net/Managers/AuthorizeNetMethod.cs

```diff
@@ -425,6 +429,7 @@ private string HmacMD5(string key, string value)
 
         private string GetDataString(NameValueCollection queryString)
         {
+            // var parameters = new[] { "x_trans_id", "x_test_request", "x_response_code", "x_auth_code", "x_cvv2_resp_code", "x_cavv_response", "x_avs_code", "x_method", "x_account_number", "x_amount", "x_company", "x_first_name", "x_last_name", "x_address", "x_city", "x_state", "x_zip", "x_country", "x_phone", "x_fa", "x_email", "x_ship_to_company", "x_ship_to_first_name", "x_ship_to_last_name", "x_ship_to_address", "x_ship_to_city", "x_ship_to_state", "x_ship_to_zip", "x_ship_to_country", "x_invoice_num" };
```

Comment:
Please ask @AliveMen on this change, why the last symbol is erased in some places.

Reply:
fixed